CanonicalLtd/documentation-builder
# Improve template

- Make use of `site_title` and `logo_url` variables (fixes #14)
- Add JavaScript to collapse side nav on small screens or doubly-nested nav lists

QA
--

Run `documentation-builder` against a repo with a `context.yaml` which contains `site_title` and `logo_url`, and see them in the built HTML.

Run `documentation-builder` against a repo with 3 levels of nav items. See deeply nested items be collapsed. And see top level items collapse on small screens.
